packages/ketcher-core/src/application/render/restruct/resgroup.js

```diff
@@ -343,9 +343,12 @@ function drawAbsoluteDat(restruct, sgroup) {
 
   const ps = sgroup.pp.scaled(options.scale)
   const name = showValue(paper, ps, sgroup, options)
-  const box = util.relBox(name.getBBox())
 
-  name.translateAbs(0.5 * box.width, -0.5 * box.height)
+  if (sgroup.data.context !== 'Bond') {
```

Comment:
I would recommend to create constants for data contexts and use it everywhere instead of string literal

Reply:
Fixed